tests/cbor_decode/test5_strange/src/main.c

```diff
@@ -241,7 +241,7 @@ void test_union(void)
 	const uint8_t payload_union2[] = {0x03, 0x23};
 	const uint8_t payload_union3[] = {0x03, 0x04};
 	const uint8_t payload_union4[] = {
-		0x65, 0x68, 0x65, 0x6c, 0x6c, 0x6f // "hello"
+		0x67, 0x22, 0x68, 0x65, 0x6c, 0x6c, 0x6f, 0x22 // "hello"
```

Comment:
Isn't this actually `g"hello"`? Is this something with CBOR that I'm missing?

Reply:
The `0x67` declares that this is a tstr with length 7. Funny that it is also an ASCII letter :).